.landscape.yaml

```diff
@@ -7,6 +7,7 @@ ignore-paths:
 test-warnings: true
 python-targets:
     - 3
+max-line-length: 200
```

Comment:
i'd say the goal is still to keep lines as short as possible but as long as necessary... the 200 is rather arbitrary to keep the noise low. i also thought about disabling it completely, would that be better to not make it look like we agreed on some number?